src/theme/DocItem/Footer/index.tsx

```diff
@@ -42,21 +55,33 @@ function EditMetaRow({
     </div>
   );
 }
-export default function DocItemFooter() {
-  const {metadata} = useDoc();
-  const {editUrl, lastUpdatedAt, formattedLastUpdatedAt, lastUpdatedBy, tags, unversionedId} =
-    metadata;
+
+export default function DocItemFooter(): JSX.Element | null {
```

Comment:
Instead of using `JSX.Element | null` you can use `React.ReactNode`.

More info https://www.totaltypescript.com/jsx-element-vs-react-reactnode

Reply:
[Take it up with Docusaurus](https://github.com/facebook/docusaurus/blob/3426f848a376ba9ee9ab680f5dd739b7cbab0f19/packages/docusaurus-theme-classic/src/theme/DocItem/Footer/index.tsx#L61) :) 

This component is actually from code written by Docusaurus (it is [swizzled](https://docusaurus.io/docs/swizzling)). We could change it, but it sounds like it's best for future upgrades to change this as little as possible.